src/components/Global/Card/index.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+
+import { Button } from '../';
+
+import styles from './mapCard.module.scss';
+
+type HARD = 'hard';
+type MEDIUM = 'medium';
+type EASY = 'easy';
+type DIFFICULTY = EASY | MEDIUM | HARD;
```

Comment:
My last review comment was meant to mean that I don't think this hassle is worthwhile. We're still doing `x === string literal` later on in the code so defining these as types doesn't really benefit us more than if we just did:
```ts
export type CardProps = {
  title: string;
  desc: string;
  diff: 'easy' | 'medium' | 'hard';
  type?: 'list' | 'grid';
};
```

Reply:
Yeah but I do this because there are some props in the functions still use the type DIFFICULTY